### ISSUE TITLE
Raised Hand .PNG displays in wrong location

### ISSUE THREAD

**Issue by ZoomTestKitchen** [bitfocus/companion-module-zoom-osc-iso#38]
Companion Version 3.0.0 (3.0.0+5617-beta-627007a4)

While viewing a Gallery Position page and raising the hand of an attendee, the Attendee Name adjusts to the top-left corner correctly, but the .png raised hand icon displays on a different button.

If another hand is raised, it will refresh the page which moves the 1st png to snap into its proper position, but the png for the 2nd raised hand is in an incorrect position.

Their are several events that trigger a refresh, such as pressing a Stream Deck button, or toggling an attendees video on/off

https://user-images.githubusercontent.com/124208331/216211908-545eeb9b-84f4-48e7-924c-84461b56b445.mp4

**Comment by digitaldrummerj** [comment]
I was able to replicate this issue.  I started a meeting on 4 computers:  2 were ZoomOSC Pro attendees, 1 was ZoomISO Pro for gallery tracking, and 1 was Zoom Client.  The Zoom client was the host.  I put ZoomISO in gallery view, pick an attendee that was not position 1, and raised my hand.  The gallery tracking buttons moved the raised hand attendee to position 1 but the raised hand image stayed in the original position of the attendee.
